packages/editor/src/components/provider/index.js

```diff
@@ -300,15 +300,7 @@ export const ExperimentalEditorProvider = withRegistryProvider(
 					}
 				);
 			}
-		}, [
-			createWarningNotice,
-			initialEdits,
-			settings,
-			post,
-			recovery,
-			setupEditor,
-			updatePostLock,
-		] );
+		}, [] );
```

Comment:
Should we add a comment indicating that dependencies are deliberately omitted?

Reply:
Sure